prime-router/src/main/resources/metadata/hl7_mapping/ORU_R01/ORU_R01-base.yml

```diff
@@ -17,53 +17,9 @@ elements:
     resource: 'Bundle.entry.resource.ofType(MessageHeader)'
     schema: classpath:/metadata/hl7_mapping/resources/MessageHeader/SFT.yml
 
-  - name: patient-information
+  - name: patient-result
     resource: 'Bundle.entry.resource.ofType(Patient)'
+    resourceIndex: patientIndex
```

Comment:
Is there a reason we can support more than one patient for the FHIR -> HL7 case?

Reply:
I was thinking if we wanted to accept batch messages with tests for multiple patients. That `patientIndex` provide the index for the patient-result group for the HL7

Reply:
Ah, but won't the condition prevent that?